x-pack/solutions/security/plugins/security_solution/public/cloud_security_posture/hooks/use_non_closed_alerts.ts

```diff
@@ -35,6 +35,12 @@ export const useNonClosedAlerts = ({
     queryId,
     to,
     from,
+    // TODO: Asset Inventory - remove temp runtime mappings
+    runtimeMappings: {
+      'related.entity': {
+        type: 'keyword',
+      },
+    },
```

Comment:
until the new fields are officially in ECS and mapped into the alerts index, we will use runtime mappings